TESTS/psa/its_ps/main.cpp

```diff
@@ -16,8 +16,8 @@
 * limitations under the License.
 */
 
-#ifndef TARGET_PSA
-#error [NOT_SUPPORTED] ITS/PS tests can run only on PSA-enabled targets.
+#if !defined(TARGET_PSA) || !defined(MBED_CONF_RTOS_PRESENT)
+#error [NOT_SUPPORTED] ITS/PS tests can run only on PSA-enabled targets and RTOS.
```

Comment:
PSA-targets with RTOS present.

Reply:
Yes PSA test cases run only in RTOS enviroment